src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs

```diff
@@ -22,7 +22,7 @@ namespace CosmosDBStudio.ViewModel.EditorTabs
     public class QuerySheetViewModel : TabViewModelBase, ISaveable
     {
         public static int UntitledCounter { get; set; }
-
+        private const string DefaultQuery = "select * from c";
```

Comment:
Could you please put this constant above the other properties and fields, separated by a new line?

Reply:
Implemented 👍

Reply:
Thanks. Could you please remove the constant, since it's been moved to QuerySheet? (actually, just undo all the changes to this file)